.github/workflows/pr-subtree-push.yml

```diff
@@ -83,15 +84,11 @@ jobs:
     steps:
       - name: Checkout Repo
         uses: actions/checkout@v4
-      - name: Setup SSH Keys
-        uses: webfactory/ssh-agent@v0.7.0
-        with:
-          ssh-private-key: |
-            ${{ secrets.APOLLO_IOS_DEV_DEPLOY_KEY }}
       - name: Configure Git
         uses: ./.github/actions/configure-git
       - name: Trigger publish-release workflow
         shell: bash
         run: |
-          gh auth login -p ssh
-          gh workflow run publish-release.yml
\ No newline at end of file
+          gh workflow run publish-release.yml
+        env:
+          GH_TOKEN: ${{ secrets.APOLLO_IOS_PAT }}
```

Comment:
Is this PAT for a general CI user or tied to one of us? If it's a personal one we should ask the security team what their preference is. _I might have asked this before, I'm not sure._

Reply:
It was created by me, so yea if we need a general use one we can switch this easy enough